tokio/src/fs/os/unix/open_options_ext.rs

```diff
@@ -0,0 +1,79 @@
+use crate::fs::open_options::OpenOptions;
+use std::os::unix::fs::OpenOptionsExt as StdOpenOptionsExt;
+
+/// Unix-specific extensions to [`fs::OpenOptions`].
+///
+/// This mirrors the definition of [`std::os::unix::fs::OpenOptionsExt`].
+///
+///
+/// [`fs::OpenOptions`]: ../../struct.OpenOptions.html
```

Comment:
This should be a intra-rustdoc link.
```suggestion
/// [`fs::OpenOptions`]: crate::fs::OpenOptions
```

Reply:
Fixed, thanks!